### PR TITLE
Make sure to work correctly with spaces in names

### PR DESCRIPTION
**Pull request by swiffer**

### CODE REVIEW

**Comment by swiffer** [comment]
ping @regisf

**Comment by regisf** [comment]
@swiffer Sorry. 
I still have an error on my PowerShell 
```Au caractère C:\Users\regis.floret\work\virtualenvwrapper-powershell\VirtualEnvWrapper.psm1:114 : 23
+     $Command = ("& '" (Join-Path (Join-Path (Split-Path $Python -Pare ...
+                       ~
Jeton inattendu « ( » dans l’expression ou l’instruction.
Au caractère C:\Users\regis.floret\work\virtualenvwrapper-powershell\VirtualEnvWrapper.psm1:114 : 22
+     $Command = ("& '" (Join-Path (Join-Path (Split-Path $Python -Pare ...
+                      ~
Parenthèse fermante « ) » manquante dans l’expression.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

```

Sorry it's in french but easily understandable (see translation upper)  

My PowerShell is 1.0.15063.0
My OS: Windows 10 Pro

Regards

Régis

File: VirtualEnvWrapper.psm1 [hunk 1]
```diff
@@ -98,7 +98,7 @@ function Invoke-CreatePyEnv($Command, $Name) {
     $NewEnv = Join-Path $WORKON_HOME $Name
     Write-Host "Creating virtual env... "
     
-    Invoke-Expression "$Command $NewEnv"
+    Invoke-Expression "$Command '$NewEnv'"
```
Comment (noc0lour) [inline VirtualEnvWrapper.psm1:101]: This change is still missing on master. If that is added spaces in folder names work for Python3 at least.

**Comment by regisf** [comment]
Sorry guys, 
I change company and as usual a lot of work. 

I don't have access to a Windows box actually. 
I will do my best to accept (or not) the PR.
But if you send me a demo that it works, I will accept it. 

Regards

Régis

**Comment by regisf** [comment]
Syntax error on line 114. Swiffer drop the repos. Partially merged with branch `fix-swiffer`

**Pull request closed without merging**